src/Sly/NotificationPusher/Adapter/Gcm.php

```diff
@@ -49,7 +49,7 @@ class Gcm extends BaseAdapter
      */
     public function supports($token)
     {
-        return (bool) preg_match('/^[0-9a-zA-Z\-\_]+$/i', $token);
+	    return is_string($token) && $token!= '';
```

Comment:
Hello,
Can you please respect basic PHP coding styles and PSR-2 standards?
No tabs but spaces (1 tab = 4 spaces), spaces before and after operators etc.
http://www.php-fig.org/psr/psr-2/fr/

I'll merge your PR after your corrections.

Thank you very much.